gatsby-config.js

```diff
@@ -131,22 +135,54 @@ const plugins = [
                   sort: { fields: [date], order: DESC }
                 ) {
                   nodes {
-                    html
+                    htmlAst
                     slug
                     title
                     date
                     description
                   }
                 }
               }
-            `,
+        `,
           serialize: ({ query: { site, allBlogPost } }) => {
+            /* This processor works on Rehype ASTs, doing two things:
+
+          1: All root-relative links are prepended with the site URL, making
+             them absolute. This uses data from siteMetadata, hence why it is
+             defined within this scope.
+
+          2: All images processed by Gatsby to be responsive are "unwrapped"
+             into their fallback 'img' nodes, as RSS doesn't work with the
+             tricks that true HTML does.
+           */
+            const processor = rehype()
+              .use(urls, url =>
+                url.href.startsWith('/')
+                  ? site.siteMetadata.site_url + url.href
+                  : undefined
+              )
+              .use(() => tree => {
+                // Unwrap all gatsby-resp-image-wrapper elements into plain images.
+                selectAll('.gatsby-resp-image-wrapper', tree).forEach(node => {
```

Comment:
should probably do something like:

```
const {
  imageClass,
  imageWrapperClass
} = require('gatsby-remark-images/constants')
```

similar to our `plugins/resize-image-plugin/index.js`

Reply:
It took me a little bit to get what you meant- good idea though! I'll do this next push.